src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -61,6 +61,11 @@
  * from Images. Images that are created from a URL, filename or byte array
  * are preloaded using MediaTracker to monitor the loaded state
  * of the image.
+ * If the image source parameter to a constructor or method is non-null,
+ * but does not reference valid accessible image data,
+ * no exceptions will be thrown but the image will be 'effectively' null,
```

Comment:
Joe asked for if there's an alternative to 'effectively' null. I'm open to suggestions. "unset" ?

Reply:
_“unset”_ is also not ideal, it's not clear what it means, especially for constructors where the image isn't set yet.

If I get it right, it refers to the fact that the image in `ImageIcon` instance is non-null, but nothing gets rendered because there's no valid image.